man/systemd.exec.xml

```diff
@@ -519,7 +519,9 @@
         <filename>/opt/</filename> hierarchies for sysext images and <filename>/etc/</filename>
         hierarchy for confext images. The order in which the images are listed will determine the
         order in which the overlay is laid down: images specified first to last will result in overlayfs
-        layers bottom to top.</para>
+        layers bottom to top. If multiple identical dm-verity protected images (i.e. with the same roothash)
+        are specified, the first one is retained and the following ones are pruned, as OverlayFS does not
+        allow duplicated dm-verity images.</para>
```

Comment:
I think this is quite misleading. overlayfs has no concept of dm-verity.

it does have a concept of a layer's fs uuid and of the fhandle of the root. And t will refuse collisions there.

Reply:
Sure, it's a high level description of the net effect - if you have the same verity device, it will be rejected. If you passed /dev/sda1 and /dev/sda1 it would reject that too, but doesn't seem likely that would be an issue for this option, no?

Reply:
corrected to say "same filesystem" instead of dm-verity image